vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -155,6 +162,17 @@ def parse_pod(pod)
         end
       end
 
+      createdby_txt = pod.metadata.annotations.try("kubernetes.io/created-by")
```

Comment:
Yes that's exactly the `try` you're referring to. I am trying to call `kubernetes.io/created-by` (too bad you can create methods with those characters) but `annotations` can be `nil`.